spec/requests/api/au/authorization_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'spec_helper'
+
+describe 'au API behavior' do
```

Comment:
こちらのPR (https://github.com/quipper/sbpayment.rb/pull/140)をマージした関係で各specのtoplevelで `RSpec.describe` としてくださいmm

```suggestion
RSpec.describe 'au API behavior' do
```

Reply:
ありがとうございます！該当箇所修正致しました。